src/ssl.c

```diff
@@ -309,8 +323,16 @@ WOLFSSL_CTX* wolfSSL_CTX_new(WOLFSSL_METHOD* method)
 void wolfSSL_CTX_free(WOLFSSL_CTX* ctx)
 {
     WOLFSSL_ENTER("SSL_CTX_free");
+#if defined(OPENSSL_EXTRA) && defined(WOLFCRYPT_HAVE_SRP) && !defined(NO_SHA256) && !defined(WC_NO_RNG)
```

Comment:
wolfSSL coding standards state that all lines should be 80 characters or less.  Please break this into 2 lines, thanks!